Client/Frontend/Browser/TabManager.swift

```diff
@@ -670,16 +672,12 @@ extension TabManager {
         let archiver = NSKeyedArchiver(forWritingWith: tabStateData)
         archiver.encode(savedTabs, forKey: "tabs")
         archiver.finishEncoding()
-        tabStateData.write(toFile: path, atomically: true)
+        let status = tabStateData.write(toFile: path, atomically: true)
+        print(status)
     }
 
     func preserveTabs() {
-        // This is wrapped in an Objective-C @try/@catch handler because NSKeyedArchiver may throw exceptions which Swift cannot handle
```

Comment:
Why don't we keep this? I think the `NSKeyedArchiver` also still throws exceptions, right? Did we not see stack traces where it threw exceptions? Better to wrap it in a handler then?